README.md

```diff
@@ -43,24 +43,35 @@ snowdrop	    https://snowdrop.github.io/helm
 To use one of the available charts, part of the snowdrop repository such as `spring-boot-example-app`, you first need to generate a chart directory using the following command:
 
 ```console
-$ helm create rest-http
+$ mkdir rest-http
 ```
-This command will generate the following directory structure using th Helm starter template:
+
+Now, let's create the `Chart.yaml` file at `rest-http/` with your Chart information:
+
+```yaml
+apiVersion: v2
+name: rest-http
+description: A Helm chart for Kubernetes
+```
+
+And let's also create the `values.yaml` file at `rest-http/` and keep it empty for now.
+
+At this point, your chart directory `rest-http` should have the following structure:
 
 ```
 rest-http
 │   Chart.yaml
 │   values.yaml
-|
-└───charts
-└───templates
-│   │   ...
 ```
 
-To update the created project using the Helm resource defined the example, we will then create the [dependencies file](https://v2.helm.sh/docs/charts/#chart-dependencies) under the root path of the project `rest-http/requirements.yaml`.
+To update the created project using the Helm resource defined the example, we will then add the [chart dependencies](https://helm.sh/docs/topics/charts/#chart-dependencies) in `rest-http/Chart.yaml`.
```

Comment:
I dont understand this text `To update the created project using the Helm resource defined the example`. To be rephrase

Reply:
Rephrased